sql/core/src/test/resources/sql-tests/results/explain.sql.out

```diff
@@ -110,6 +114,10 @@ struct<plan:string>
 
 (1) Scan parquet default.explain_temp1 
 Output: [key#x, val#x]
+Batched: true
+Location [not included in comparison]
+PushedFilters: [IsNotNull(key), GreaterThan(key,0)]
```

Comment:
do we have a test case to display the DPP filter?

Reply:
@cloud-fan I have enhanced the test in ExplainSuite to test the DPP filter. What do you think ?

Reply:
I'm asking it because I don't see where we extract the DPP filter in this PR. You can see from `FileSourceScanExec#dynamicallySelectedPartitions` that we can get DPP filter by `partitionFilters.filter(isDynamicPruningFilter)`.